cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-kind: ReplicationController
+apiVersion: apps/v1beta1
+kind: StatefulSet
```

Comment:
StatefulSet is a very specialized form of deploying pods, i.e. with identity for pods and pv management for this setup. It feels pretty odd to use a StatefulSet if both features are not needed. I would very much prefer to use a Deployment instead here if these assumptions are right.